tests/utils/svelte-component.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+  // Check for TS support
+	let count: number = 0;
+
+	function handleClick() {
+		count += 1;
+	}
+</script>
+
+<button on:click={handleClick}>
+	Clicked {count} {count === 1 ? 'time' : 'times'}
+</button>
+
+<style lang="scss">
+$bg: red;
+
+button {
+  background: $bg;
+}
+</style>
```

Comment:
Nit, can you adjust the tabbing in this file, it's all over the place. 😅

Reply:
Oh yeah good point, this is what I get for copy-pasting svelte examples from a bunch of places 😂